### ISSUE TITLE
What is the password of jenkins user inside docker container?

### ISSUE THREAD

**Issue by Celebes** [jenkinsci/docker#1003]
I have Windows 10 Home with Docker desktop installed using WSL2. I am trying to setup a CI/CD pipeline that will be able to build docker images. To avoid using docker in docker (dind) I created a volume "**/var/run/docker.sock:/var/run/docker.sock**". Then I created a custom Dockerfile for my jenkins container like this, to install docker client without docker engine/daemon inside container:

```
FROM jenkins/jenkins
USER root
RUN apt-get update && \
	apt-get -y install sudo && \
	curl -sfL -o docker.tgz "https://download.docker.com/linux/static/stable/x86_64/docker-19.03.13.tgz" && \
	tar -xzf docker.tgz --directory /usr/local/bin && \
	rm docker.tgz && \
	groupadd docker && \
	usermod -aG docker jenkins && \
	newgrp docker
ENV PATH="${PATH}:/usr/local/bin/docker"
USER jenkins
```

The problem I have now is error when trying to build an image using jenkins:

```
+ docker build -t my-img .
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.40/build?buildargs=%7B%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&session=x0hnv61ahjx3k69qa96fle3eh&shmsize=0&t=registry%3A5000%2Fcalculator&target=&ulimits=null&version=1: dial unix /var/run/docker.sock: connect: permission denied
```

I tried to avoid it by issuing command `sudo chmod 666 /var/run/docker.sock` but it requires password for default container user (jenkins) and I can't find it anywhere.

**tl;dr - the default user inside docker container is "jenkins", what is his password?**

So far I tried:
* 1234
* admin
* jenkins

But none of them worked

**Comment by slide** [comment]
There is no password set by any of the scripts. You would need to set the password as part of your inherited dockerfile with something like:

```
USER root
RUN echo "linuxpassword" | passwd --stdin linuxuser
``` 

if your version of Linux support it, or something like if it doesn't:

```
USER root
RUN echo -e "linuxpassword\nlinuxpassword" | passwd linuxuser
```

Also, why not do the chmod in your existing RUN that is already running as root?
